### ISSUE TITLE
Multi-Monitor Support

### ISSUE THREAD

**Issue by kingy444** [jonaskohl/CapsLockIndicator#61]
Notifications currently stick to the screen the mouse is on, would like an option to display on all monitors

**Comment by jonaskohl** [comment]
Good point! I will add this (keep in mind though that this would have to be enabled manually, because showing more overlay windows means using up more system resources, albeit a miniscule amount)

**Comment by kingy444** [comment]
just adding a comment to keep this one alive - any chance we could add this soon 👍

**Comment by kingy444** [comment]
> 

Just me keeping this active - thanks
